android/app/src/main/java/com/onair/hearit/presentation/search/SearchViewModel.kt

```
@@ -25,6 +26,10 @@ class SearchViewModel(
     private val _toastMessage = SingleLiveData<Int>()
     val toastMessage: LiveData<Int> = _toastMessage
 
+    private lateinit var paging: Paging
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)